scripts/generate-repos-info.js

```diff
@@ -0,0 +1,71 @@
+const fs = require("fs");
+const axios = require("axios");
+const listr = require("listr")
+
+axios.defaults.timeout = 5000;
+
+const axiosConfig = {
+  headers: {
+    "content-type": "application/json",
+    Accept: "application/vnd.github.v3+json"
+  },
+}
+
+const repoList = require("../website/config/docs").map(v => v.githubRepo)
+const res = {}
+
+const tasks = new listr([
+  {
+    title: "Fetch repos' info and good first issues",
+    task: ()=> new listr(repoList.map((repo)=>{
+      return {
+        title: `Fetch ${repo}`,
+        task: ()=>new listr([
```

Comment:
Formatting code required.

Reply:
Sorry, do we have any lint?

Reply:
We don't have a lint script for javascript scripts yet, so you can rely on the IDE's code formatting features for now.